lib/game.dart

```diff
@@ -314,50 +313,45 @@ class SimpleGame extends BaseGame {
 /// Provided it with a [Game] instance for your game and the optional size of the widget.
 /// Creating this without a fixed size might mess up how other components are rendered with relation to this one in the tree.
 /// You can bind Gesture Recognizers immediately around this to add controls to your widgets, with easy coordinate conversions.
-class EmbeddedGameWidget extends StatefulWidget {
+class EmbeddedGameWidget extends LeafRenderObjectWidget{
   final Game game;
   final Position size;
-
   EmbeddedGameWidget(this.game, {this.size});
 
-  @override
-  State<StatefulWidget> createState() {
-    return _EmbeddedGameWidgetState();
-  }
-}
-
-class _EmbeddedGameWidgetState extends State<EmbeddedGameWidget> {
-  _EmbeddedGameWidgetState();
 
   @override
-  void initState() {
-    super.initState();
-    WidgetsBinding.instance.addPostFrameCallback(_afterLayout);
-  }
-
-  @override
-  void didUpdateWidget(EmbeddedGameWidget oldWidget) {
-    super.didUpdateWidget(oldWidget);
-    WidgetsBinding.instance.addPostFrameCallback(_afterLayout);
-  }
+  RenderBox createRenderObject(BuildContext context) {
+    if(size != null){
+      return RenderEmbeddedGameBox(
+          _GameRenderBox(context, game),
+          widgetSize: this.size
+      );
+    }
+    return _GameRenderBox(context, game);
 
-  void _afterLayout(_) {
-    RenderBox box = context.findRenderObject();
-    widget.game.builder.offset = box.localToGlobal(Offset.zero);
   }
 
   @override
-  Widget build(BuildContext context) {
-    if (widget.size == null) {
-      return widget.game.widget;
+  void updateRenderObject(BuildContext context, RenderBox renderObject) {
+    if(size != null){
+      RenderEmbeddedGameBox _renderObject = renderObject as RenderEmbeddedGameBox;
+      _renderObject..game = _GameRenderBox(context, game)..widgetSize = this.size;
+    } else {
+      _GameRenderBox _renderObject = renderObject as _GameRenderBox;
+      _renderObject..game = game;
```

Comment:
here you can use only a single dot, right?

Reply:
![image](https://user-images.githubusercontent.com/6718144/54749607-db2e8900-4bb3-11e9-8ff8-9d2d3d4b14d5.png)